docs/REST API/rest-api.md

```diff
@@ -7229,45 +7357,130 @@ Deletes the session specified by ID. When the user associated with the session n
         "hosts_count": 1
       },
       {
-        "id": 2,
+        "id": 32,
         "name": "1Password – Password Manager",
-        "version": "2.3.7",
+        "versions": "2.10.0",
```

Comment:
@marko-lisica I see this is `versions` in the previous software item but was updated to `versions` here — which is correct? (Guessing `version` since it's not an array?)

Reply:
@rachaelshaw good catch. This example is for the `GET /api/v1/fleet/software/versions` endpoints which returns a single `version` for each item in the list.

Updated this to singular `version`.